pkg/util/util.go

```diff
@@ -23,19 +21,19 @@ const (
 var ChartNamespace string
 
 func GetEncryptionTransformers(encryptionConfigSecretName string, secrets v1core.SecretController) (map[schema.GroupResource]value.Transformer, error) {
-	var transformerMap map[schema.GroupResource]value.Transformer
+	// var transformerMap map[schema.GroupResource]value.Transformer
 	// EncryptionConfig secret ns is hardcoded to ns of controller in chart's ns
 	// kubectl create secret generic test-encryptionconfig --from-file=./encryption-provider-config.yaml
 	logrus.Infof("Get encryption config from namespace %v", ChartNamespace)
-	encryptionConfigSecret, err := secrets.Get(ChartNamespace, encryptionConfigSecretName, k8sv1.GetOptions{})
-	if err != nil {
-		return transformerMap, err
-	}
-	encryptionConfigBytes, ok := encryptionConfigSecret.Data[encryptionProviderConfigKey]
-	if !ok {
-		return transformerMap, fmt.Errorf("no encryptionConfig provided")
-	}
-	return encryptionconfig.ParseEncryptionConfiguration(bytes.NewReader(encryptionConfigBytes))
+	// encryptionConfigSecret, err := secrets.Get(ChartNamespace, encryptionConfigSecretName, k8sv1.GetOptions{})
```

Comment:
just remove this?

Reply:
@superseb yes, Just wanted to test that everything still works correctly before removing.